editor/editor_about.cpp

```diff
@@ -161,7 +161,8 @@ EditorAbout::EditorAbout() {
 
 	Label *about_text = memnew(Label);
 	about_text->set_v_size_flags(Control::SIZE_SHRINK_CENTER);
-	about_text->set_text(String::utf8("\xc2\xa9 2014-present ") + TTR("Godot Engine contributors") + "." +
+	about_text->set_text(
+			String::utf8("\xc2\xa9 2014-present ") + TTR("Godot Engine contributors") + "." +
```

Comment:
I know it's been like that before, but I find it weird that we translate that part specifically, but not, say, "present". 🙃

Reply:
Also dates themselves are subject to translation

Reply:
We could create a PR afterwards to translate that part of the string.